src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -267,6 +270,16 @@ function MetadataSidebarRedesign({
     const taxonomyNodeFetcher = async (scope: string, taxonomyKey: string, nodeID: string) =>
         metadataTaxonomyNodeAncestorsFetcher(api, fileId, scope, taxonomyKey, nodeID);
 
+    useEffect(() => {
```

Comment:
This effect is causing that createSessionRequest is being triggered multiple times (probably due to createSessionRequest method signature change. Can we add a flag to control when the request state is ongoing to avoid triggering this multiple times maybe?

Reply:
If `createSessionRequest` is defined inside a component it will always be created a new function unless it's inside a `React.useCallback`